protocol-designer/src/localization/en/form.json

```
@@ -83,7 +83,13 @@
         "label": "delay"
       },
       "tip_position": { "label": "tip position" },
-      "flow_rate": { "label": "Flow Rate" },
+      "flow_rate": {
+        "label": "Flow Rate",
+        "range": "between {{min}} and {{max}}",
+        "error_decimals": "A max of {{decimals}} decimal places is allowed",
+        "error_out_of_bounds": "accepted range is {{min}} to {{max}}",
+        "default_text": "Our default aspirate speed is optimal for a {{displayName}} aspirating liquids with a viscosity similar to water"
```

We should either 
- customize this text depending on whether the user is editing an aspirate or dispense rate
- make it generic and use the term "flow rate"

i think i prefer the latter. it's more consistent with PAPI docs. so, maybe something like this:
```suggestion
        "default_text": "The default {{displayName}} flow rate is optimal for handling aqueous liquids"
```

---

thanks Ed! i copied over the text that was there into i18n. This is a lot better!